spacy_llm/backends/integration/hf/dolly.py

```diff
@@ -1,19 +1,20 @@
-from typing import Iterable, Callable, Any, Dict
+from typing import Any, Callable, Dict, Iterable
 
-from spacy.util import SimpleFrozenList, SimpleFrozenDict
+from spacy.util import SimpleFrozenDict, SimpleFrozenList
 
-
-from . import HuggingFaceBackend
 from ....compat import transformers
 from ....registry.util import registry
+from . import HuggingFaceBackend
 
 
 class DollyBackend(HuggingFaceBackend):
     def init_model(self) -> Any:
         """Sets up HF model and needed utilities.
         RETURNS (Any): HF model.
         """
-        return transformers.pipeline(model=self._model_name, **self._config)
+        return transformers.pipeline(
+            model=self._model_name, **self._config.get("init", {})
```

Comment:
Here we could use `self._init_config` instead

Reply:
Resolving this, let's continue in https://github.com/explosion/spacy-llm/pull/151#discussion_r1213009097.